src/test/java/apoc/load/XmlTest.java

```diff
@@ -138,4 +138,81 @@ public void testReturnCollectionElements() {
                 });
     }
 
+    @Test
+    public void testReturnCollectionElementsWithPathTitle() {
+        testResult(db, "call apoc.load.xml('file:src/test/resources/books.xml', '/catalog/book[genre=\"Computer\"]/title') yield value as result\n" +
+                        "WITH result._text as title\n" +
+                        "RETURN  title"
+                , result -> {
+                    assertEquals("+------------------------------------------+\n" +
```

Comment:
can we please not use that? we're tying ourselves to the tabular format.
but instead like in other places check the result's columns and then the values.
you can also in the statement order and collect so you only have to check a single collection and can use `testCall`